meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/u-boot/patches/0003-Integration-of-Mender-boot-code-into-U-Boot.patch

```diff
@@ -0,0 +1,51 @@
+From de38de98f3032825d1553e83fa6957d7438bbdc4 Mon Sep 17 00:00:00 2001
```

Comment:
I think we can safely remove all uboot patches and recipes from Jetpack 5 as uboot is no longer a part of Jetpack 5 support